src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj

```diff
@@ -32,6 +32,9 @@
 
   <ItemGroup>
     <PublishReadyToRunCrossgen2ExtraArgsList Include="--compositekeyfile:$(AssemblyOriginatorKeyFile)"/>
+    <!-- Compile with avx2 on x64 -->
+    <PublishReadyToRunCrossgen2ExtraArgsList Condition="'$(TargetArchitecture)' == 'x64'" Include="--inputbubble"/>
+    <PublishReadyToRunCrossgen2ExtraArgsList Condition="'$(TargetArchitecture)' == 'x64'" Include="--instruction-set:avx2"/>
```

Comment:
Nit: we normally put a space between `"` and `/>` in XML files.

Reply:
fixed.